include/NovelRT/Ecs/Configurator.h

```diff
@@ -85,6 +87,23 @@ namespace NovelRT::Ecs
             return *this;
         }
 
+        /**
+         * @brief Configures systems for registration for this ECS instance (IEcs variant).
```

Comment:
```suggestion
         * @brief Configures systems for registration for this ECS instance (IEcsSystem variant).
```
We do this to allow better doxygen linkage.

Reply:
Done.